tests/torch/sparsity/movement/test_algo.py

```diff
@@ -0,0 +1,348 @@
+                        count_sparse_modules += 1
+                assert count_sparse_modules > 0
+
+    run_movement_pipeline(tmp_path, compression_ctrl, compressed_model, [CheckBinaryMaskCallback(compression_ctrl)])
+
+
+# This part is not finished. How to check sparisty with fill stage?
+@pytest.mark.parametrize('nncf_config_builder', [
+    ConfigBuilder(),
+])
+def test_sparsity_statistics_are_increasing(tmp_path, nncf_config_builder):
+    nncf_config = nncf_config_builder.build(log_dir=tmp_path)
+    compression_ctrl, compressed_model = create_compressed_model(bert_tiny_torch_model(), nncf_config)
+    assert isinstance(compression_ctrl.statistics().movement_sparsity, MovementSparsityStatistics)
+    log_dict = defaultdict(list)
+    compressed_model.train()
+    for epoch in range(10):
+        compression_ctrl.scheduler.epoch_step()
+        for batch in range(4):
+            compression_ctrl.scheduler.step()
+            statistics = compression_ctrl.statistics().movement_sparsity
+            log_dict['importance_threshold'].append(statistics.importance_threshold)
+            log_dict['importance_regularization_factor'].append(statistics.importance_regularization_factor)
+            log_dict['sparsity_level'].append(statistics.model_statistics.sparsity_level)
+            log_dict['sparsity_level_for_layers'].append(statistics.model_statistics.sparsity_level_for_layers)
+
+    def is_non_decreasing(x: List):
+        return all(a <= b for a, b in zip(x[:-1], x[1:]))
+
+    assert is_non_decreasing(log_dict['importance_threshold'])
+    assert is_non_decreasing(log_dict['importance_regularization_factor'])
+    assert is_non_decreasing(log_dict['sparsity_level'])
+    assert is_non_decreasing(log_dict['sparsity_level_for_layers'])
+
+
+def get_linear_weight_bias_data(module: NNCFLinear):
+    # TODO: how can we get this via UpdateWeightAndBias?
+    in_features = module.in_features
+    zero_input = torch.zeros((1, in_features))
+    eye_input = torch.eye(in_features)
+    with torch.no_grad():
+        bias = module(zero_input)
+        weight = module(eye_input) - bias
+    return weight.T, bias
+
+
+def check_onnx_has_sparsified_param(compressed_model, compression_ctrl, onnx_path):
+    compressed_model.eval()
+    ref_params = {}
+    module_name_dict = {module: name for name, module in compressed_model.named_modules()}
+    for m in compression_ctrl.sparsified_module_info:
+        weight, bias = get_linear_weight_bias_data(m.module)
+        name = module_name_dict[m.module]
+        ref_params[name + '.weight'] = weight
+        ref_params[name + '.bias'] = bias
+
+    # temporary solution to preserve param names in onnx model
+    _original_export_fn = torch.onnx.export
+    _tmp_export_fn = partial(_original_export_fn, do_constant_folding=False)
+    torch.onnx.export = _tmp_export_fn
+    compression_ctrl.export_model(onnx_path)
+    torch.onnx.export = _original_export_fn
```

Comment:
Seems like support for more customized export will come later with new unified NNCF API.
As for now, patching for the test should be OK, but I'd recommend doing it via test utility:
```
    from unittest.mock import patch
    with patch("torch.onnx.export", wraps=partial(torch.onnx.export, do_constant_folding=False)):
        compression_ctrl.export_model(onnx_path)
```